src/mixins/keycloak.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  computed: {
+    keycloakLoginRedirect() {
+      let redirect;
+      if (this.$route) {
+        if (this.$route.query && this.$route.query.redirect) {
+          redirect = this.$route.query.redirect;
+        } else if (this.$route.fullPath) {
+          redirect = this.$route.fullPath;
+        }
+      }
+      return redirect;
+    },
+
+    keycloakAccountUrl() {
+      const keycloakAccountUrl = new URL(
+        `/auth/realms/${this.$auth.strategy.options.realm}/account`, this.$auth.strategy.options.origin
+      );
+      keycloakAccountUrl.search = new URLSearchParams({
+        referrer: this.$auth.strategy.options.client_id,
+        'referrer_uri': this.$config.app.baseUrl
```

Comment:
The baseUrl is passed in here as the referrer uri, which I believe is always set to production, but shouldn't we want this to be more dynamic and it linking back to test or localhost when navigating from there?

Reply:
It's set by env var `PORTAL_BASE_URL`, so if you want it to be your localhost URL, set it to be so. :smiley: